src/main/java/entity/Attachment.java

```diff
@@ -0,0 +1,82 @@
+package entity;
+
+import org.springframework.web.multipart.MultipartFile;
+
+import org.springframework.cloud.gcp.data.spanner.core.mapping.PrimaryKey;
+import org.springframework.cloud.gcp.data.spanner.core.mapping.Column;
+import org.springframework.cloud.gcp.data.spanner.core.mapping.Table;
+import org.springframework.cloud.gcp.data.spanner.core.mapping.NotMapped;
+
+@Table(name = "Attachment")
+public final class Attachment {
+    
+    @PrimaryKey
+    @Column(name = "AttachmentID")
+    private long attachmentId;
+
+    @Column(name = "FileName")
+    private String fileName;
+
+    @Column(name = "FileType")
+    private String fileType;
+
+    @Column(name = "FileSize")
+    private long fileSize;
```

Comment:
This lacks some info. What is the unit of this?
We should include the unit it either the type or name.

Reply:
Sorry, this wasn't clear to me. Did you mean I should add Javadoc mentioning the unit of this variable?

Reply:
Done